oxd-server/src/main/java/org/gluu/oxd/server/persistence/modal/RpObject.java

```diff
@@ -0,0 +1,59 @@
+package org.gluu.oxd.server.persistence.modal;
+
+import java.io.Serializable;
+
+import org.gluu.persist.annotation.AttributeName;
+import org.gluu.persist.annotation.DN;
+import org.gluu.persist.annotation.DataEntry;
+import org.gluu.persist.annotation.ObjectClass;
+
+@DataEntry
+@ObjectClass("oxRp")
+public class RpObject implements Serializable {
+
+    @DN
+    private String dn;
+    @AttributeName(name = "id")
+    private String id;
+    @AttributeName(name = "data")
```

Comment:
It should have `@JsonObject` annotation to indicate that it's json, not string.

Reply:
Done